ui/urls.py

```diff
@@ -12,74 +13,76 @@
 router.register(r"subtitles", views.VideoSubtitleViewSet, basename="subtitle")
 
 urlpatterns = [
-    url(r"^$", views.index, name="index"),
-    url(r"^login/$", views.LoginView.as_view(), name="login"),
-    url(r"^logout/$", LogoutView.as_view(next_page=settings.LOGIN_URL), name="logout"),
-    url(
+    path("", views.index, name="index"),
+    path("login/", views.LoginView.as_view(), name="login"),
+    path("logout/", LogoutView.as_view(next_page=settings.LOGIN_URL), name="logout"),
+    re_path(
         r"^collections/(?P<collection_key>[0-9a-f]{32})?/?$",
         views.CollectionReactView.as_view(),
         name="collection-react-view",
     ),
-    url(
+    re_path(
         r"^collections/[0-9A-Za-z\-_\:]+/videos/(?P<video_key>\d+)(-.+)?/?$",
         views.TechTVDetail.as_view(),
         name="techtv-collection-detail",
     ),
-    url(r"^help/", views.HelpPageView.as_view(), name="help-react-view"),
-    url(r"^terms/", views.TermsOfServicePageView.as_view(), name="terms-react-view"),
-    url(
+    re_path(r"^help/", views.HelpPageView.as_view(), name="help-react-view"),
+    re_path(
+        r"^terms/", views.TermsOfServicePageView.as_view(), name="terms-react-view"
+    ),
+    re_path(
         r"^videos/(?P<video_key>[0-9a-f]{32})/$",
         views.VideoDetail.as_view(),
         name="video-detail",
     ),
-    url(
+    re_path(
         r"^videos/(?P<video_key>[0-9a-f]{32})/embed/$",
         views.VideoEmbed.as_view(),
         name="video-embed",
     ),
-    url(
+    re_path(
         r"^videos/(?P<video_key>[0-9a-f]{32})/download",
         views.VideoDownload.as_view(),
         name="video-download",
     ),
-    url(
+    re_path(
         r"^videos/(?P<video_key>\d+)(-.+)?/?$",
         views.TechTVDetail.as_view(),
         name="techtv-detail",
     ),
-    url(
+    re_path(
         r"^videos/(?P<video_key>.+)/private/?$",
         views.TechTVPrivateDetail.as_view(),
         name="techtv-private",
     ),
-    url(
+    re_path(
         r"^videos/(?P<video_key>\d+)(-.+)?/download",
         views.TechTVDownload.as_view(),
         name="techtv-download",
     ),
-    url(
+    re_path(
         r"^embeds/(?P<video_key>\d+)(-.+)?/?$",
         views.TechTVEmbed.as_view(),
         name="techtv-embed",
     ),
-    url(r"^transcode/", include("dj_elastictranscoder.urls")),
-    url(
-        r"^api/v0/upload_videos/$",
+    # path("transcode/", include("dj_elastictranscoder.urls")),
```

Comment:
We have commented this as we are replacing transcoder `elastictranscoder` with `MediaConvert`
@umar8hassan will integrate that in a different PR